config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -16,8 +16,8 @@
 import math
 import json
 import re
-import jinja2 # pylint: disable=import-error
 import yaml
+import jinja2 # pylint: disable=import-error
```

Comment:
should we ensure the linter _can_ actually import? I think this just means we didn't install deps before linting?

Reply:
doesn't have much context here, but seems like this package is imported in a weird way hack/run-in-python-container.sh

Reply:
It doesn't look to me like we install deps before linting?

Reply:
sorry for the confusion, I had a bad copy/paste in previous response, meant to link https://github.com/kubernetes/test-infra/blob/b5865074c4ed1cfb41f9e455e9860a6a4e60794b/gubernator/lint.sh#L17, package was linted with a separate script, and I'm not sure whether we want to incorporate this logic globally

Reply:
Er for gubernator I think this is because we're not handling importing it outside of GAE magic 
https://github.com/kubernetes/test-infra/blob/f7e21a3c18f4f4bbc7ee170675ed53e4544a0632/gubernator/app.yaml#L39

which feels bad, I'm not sure how kops was related though ..?

Reply:
I still think this should be fixed, but leaving to a follow-up. we should poke one of the kops owners at least?